### PR TITLE
VARCem builds with default project now.

### PR DESCRIPTION
**Pull request by MoochMcGee**

### CODE REVIEW

**Comment by waltje** [comment]
Handled manually. Thanks!

**Comment by MoochMcGee** [comment]
Weird, because you never committed it.

File: src/win/msvc/vc15/VARCem.vcxproj [hunk 1]
```diff
@@ -668,6 +668,8 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">
     <LinkIncremental>false</LinkIncremental>
+    <IncludePath>$(ProjectDir)\..\include;$(ProjectDir)\..\..\mingw\include;$(ProjectDir)\..\..\mingw\include\pcap;$(IncludePath)</IncludePath>
+    <LibraryPath>$(ProjectDir)\..\lib\x86;$(VC_LibraryPath_x86);$(WindowsSDK_LibraryPath_x86);$(NETFXKitsDir)Lib\um\x86</LibraryPath>
```
Comment (darkstar) [inline src/win/msvc/vc15/VARCem.vcxproj]: the NetFX path is not needed. 
Also, the include/lib paths are set in the global.props file. Please add them there, then they're included in both the debug and release builds

File: src/win/msvc/vc15/VARCem.vcxproj [hunk 2]
```diff
@@ -659,6 +659,8 @@
   <PropertyGroup Label="UserMacros" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <LinkIncremental>true</LinkIncremental>
+    <IncludePath>$(ProjectDir)\..\include;$(ProjectDir)\..\..\mingw\include;$(ProjectDir)\..\..\mingw\include\pcap;$(IncludePath)</IncludePath>
+    <LibraryPath>$(ProjectDir)\..\lib\x86;$(VC_LibraryPath_x86);$(WindowsSDK_LibraryPath_x86);$(NETFXKitsDir)Lib\um\x86</LibraryPath>
```
Comment (darkstar) [inline src/win/msvc/vc15/VARCem.vcxproj]: Add the files to the global.props file. And the NetFx path (and the others) are not needed

File: src/win/msvc/vc15/VARCem.vcxproj.user [hunk 3]
```diff
@@ -1,4 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
+    <LocalDebuggerWorkingDirectory>$(ProjectDir)\..\..\..\..\</LocalDebuggerWorkingDirectory>
+    <DebuggerFlavor>WindowsLocalDebugger</DebuggerFlavor>
+  </PropertyGroup>
```
Comment (darkstar) [inline src/win/msvc/vc15/VARCem.vcxproj.user]: Is there a reason this change is in here too? If not, remove it from the commit please

**Pull request closed without merging**